nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/RequestTracingNotificationEventFactory.java

```diff
@@ -44,6 +45,12 @@ NotificationEvent build(long elapsedTime, NotifierType notifierType) {
             event.setMessage(getRequestEventsAsStr());
             return event;
         }
+        if (NotifierType.HIPCHAT.equals(notifierType)) {
```

Comment:
I don't like this if type equals code in this class as this ties Request Tracing to HipChat. Shouldn't request tracing just send a notification not caring what types of notifiers exist or are configured ?